docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,28 +313,69 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+	ShutdownProperly := func(ctx context.Context, hz *hazelcast.Client, err error) {
```

Comment:
I believe we can make a "defer" call right after handling returned error values of "hazelcast.StartNewClientWithConfig". Such as:
`defer hz.Shutdown(ctx)`

If we do that, we can handle following errors simply by
`if err != nil {
log.Fatal(err)
}`

Reply:
What if `hz.Shutdown(ctx)` return an error ?

Reply:
Clients are generally shutdown at the end of the programs or some scope. Meaning that resources are cleared even if you do not act on the returned error. One use case I can think of is opening/closing multiple clients, which would create some leaks if error is not handled.

But generally handling error on shutdown is not critical for majority of cases as you can see from examples in go client. If you really want to handle it somehow I would suggest
```
defer func() {
        if err := client.Shutdown(ctx); err != nil {
            log.Println("could not shutdown client gracefully")	
        }
}()
```
or some variation of it. How does it sound?

Reply:
Sounds good to me!